napari/_tests/test_view_layers.py

```diff
@@ -42,7 +42,7 @@ def test_docstring(layer):
     # check summary section
     method_summary = ' '.join(method_doc['Summary'])  # join multi-line summary
 
-    summary_format = 'Add an? .+? layer to the layer list.'
+    summary_format = 'Add an? .+? layers? to the layer list.'
```

Comment:
The test failure comes from changing "Add an image layer to the layer list." to "Add one or more Image layers to the layer list"

Reply:
On mobile: I thought I made a pr with the fix to your branch.

Reply:
https://github.com/Czaki/napari/pull/36
I do not know why I do not get notifications